Networking/NetworkingTests/Mapper/RefundMapperTests.swift

```diff
@@ -61,6 +61,64 @@ final class RefundMapperTests: XCTestCase {
         XCTAssertEqual(item.sku, "HOODIE-SHIP-YOUR-IDEA-BLUE-XL")
         XCTAssertEqual(item.price, NSDecimalNumber(integerLiteral: 27))
     }
+
+    func test_refund_is_encoded_correctly_with_items_and_taxes() throws {
+        // Given
+        let refund = sampleRefund(includeTaxes: true)
+        let mapper = RefundMapper(siteID: dummySiteID, orderID: 123)
+
+        // Ref: https://github.com/woocommerce/woocommerce/blob/0a81188e42fd3797ea2dd09f35c777e787801808/includes/wc-order-functions.php#L535-L573
```

Comment:
I suggest to use a short link here 👍 
```suggestion
        // Ref: https://git.io/JTRsF
```